### PR TITLE
add(pr template)

### PR DESCRIPTION
**Pull request by ChrOertlin**
### How to test

- [ ] Deploy your janus branch on `cg-services-stage` - Follow the atlas container documentation.
- [ ] Make sure the newly added functionality works by using the swagger: janus-stage.sys.scilifelab.se
- [ ] Make sure the command `cg store qc-metrics` can be run to completion on `hasta` stage.

## Implementation Plan

- [ ] Understand and follow the steps outline in CONTRIBUTING.md
- [ ] Make sure that any `model` additions are reflected in the `arnold` repository https://github.com/Clinical-Genomics/arnold

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Description
+
+
+### How to test
+
+- [ ] Deploy your janus branch on `cg-services-stage` - Follow the atlas container documentation.
+- [ ] Make sure the newly added functionality works by using the swagger: janus-stage.sys.scilifelab.se
```
Comment (islean) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: If we have a swagger-ui it would be nice to link it here

**Review by islean — APPROVED** [review]
Looks good! Might want to add the ###Added, ###Fixed etc if you like that structure.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ChrOertlin) [inline .github/PULL_REQUEST_TEMPLATE.md:7]:
```suggestion
- [ ] Make sure the newly added functionality works by using the swagger: https://janus-stage.sys.scilifelab.se/
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+## Description
+
+
+### How to test
+
+- [ ] Deploy your janus branch on `cg-services-stage` - Follow the atlas container documentation.
```
Comment (ChrOertlin) [inline .github/PULL_REQUEST_TEMPLATE.md:6]:
```suggestion
- [ ] Deploy your janus branch on `cg-services-stage` - Follow the atlas container documentation (https://atlas.scilifelab.se/infrastructure/containers/).
```

**Pull request merged**